go-corona-app/screens/HomeScreen.js

```diff
@@ -1,138 +1,121 @@
 import * as WebBrowser from 'expo-web-browser';
-import * as React from 'react';
-import { Image, Platform, StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import React, { Component } from 'react';
+import { Image, Platform, Dimensions, StyleSheet, Text, TouchableOpacity, View, ActivityIndicator } from 'react-native';
 import { ScrollView } from 'react-native-gesture-handler';
+import MapView, { PROVIDER_GOOGLE, Heatmap } from 'react-native-maps';
+import Constants from 'expo-constants';
+import * as Location from 'expo-location';
+
+import data from '../test_data/location_history.js'
+
 
 import { MonoText } from '../components/StyledText';
+import { renderer } from 'react-test-renderer';
```

Comment:
@anjaneyulukinnara - any idea why we need a test renderer?

Reply:
no need, I used to write in typescript which shows unused imports, not the case here will remove in next pull